README.md

```diff
@@ -51,7 +51,7 @@ The following dependencies are required:
 
 - TypeScript >=4.6
 - Node 16+
-- Angular 13+
+- Angular 15+
 
 ## Installation
 
```

Comment:
@TheUnderScorer Could you add section on linkedIds as part of this PR? https://fingerprintjs.atlassian.net/browse/INTER-586

Reply:
Yep, done!